builder/deltapacks.go

```diff
@@ -81,11 +81,12 @@ func createDeltaPacks(fromMoM *swupd.Manifest, toMoM *swupd.Manifest, printRepor
 					continue
 				}
 
+				report := fmt.Sprintf("  Finished delta pack for bundle %q from %d to %d\n", b.Name, b.FromVersion, b.ToVersion)
 				if len(info.Warnings) > 0 {
 					for _, w := range info.Warnings {
-						fmt.Printf("    WARNING: %s\n", w)
+						report += fmt.Sprintf("    WARNING: %s\n", w)
```

Comment:
I think it makes more sense to separate warnings from the report by writing them to stderr. @reaganlo Thoughts?

Reply:
Makes sense.

Reply:
I updated the PR so that warning messages include the impacted bundle's name and are written to stderr.